dddwa/dddperth-website
# 2021 Sponsors

Adds the data and images for the 2021 sponsors

### Testing
* View http://localhost:3000/
* Scroll down to the sponsors and you should see all the logos
* They should go to the correct url
